### PR TITLE
RELATED: RAIL-3064 use measure names from execution response on tiger

### PR DESCRIPTION
**Pull request by no23reason**
As part of this additional sanity in the tiger client was done:

* the tiger client was re-generated -> this made some type changes that I adapted to in the rest of the codebase
* new endpoints from the `/api/actions` were used instead of the old equivalents
* the execution part of api-client-tiger now uses the generated controllers (as do the other endpoints) - this needed a break of the API - it now returns `AxiosPromise` like the rest of the endpoints

Since the api-client-tiger is broken as is, I think this breaking change is ok now.

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2869/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4499/ : SUCCESS in 12m 31s

File: libs/sdk-backend-tiger/src/backend/workspace/execution/executionResult.ts [hunk 1]
```diff
@@ -67,7 +67,12 @@ export class TigerExecutionResult implements IExecutionResult {
 
     public async readAll(): Promise<IDataView> {
         const executionResultPromise = this.authCall((client) =>
-            client.execution.executionResult(this.workspace, this.resultId),
+            client.executionResult
+                .retrieveResult({
+                    workspaceId: this.workspace,
+                    resultId: this.resultId,
+                })
+                .then(({ data }) => data),
```
Comment (lupko) [inline libs/sdk-backend-tiger/src/backend/workspace/execution/executionResult.ts:75]: not related to this PR but we rather double-check whether the call without paging actually returns all the data or some first-default-page only. if the latter is the case we have a bug.

Comment (no23reason) [reply libs/sdk-backend-tiger/src/backend/workspace/execution/executionResult.ts:75]: hm, I'll try to check it in AD, good point

**Review by lupko — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1133

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/846/ : SUCCESS in 38s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1110/ : SUCCESS in 27s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4511/ : SUCCESS in 16m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/220/ : SUCCESS in 9m 54s (non-voting)

**Pull request merged**